internal/ethapi/api.go

```diff
@@ -80,28 +80,28 @@ func (s *PublicEthereumAPI) MaxPriorityFeePerGas(ctx context.Context) (*hexutil.
 	return (*hexutil.Big)(tipcap), err
 }
 
-type feeHistoryResults struct {
-	FirstBlock   rpc.BlockNumber
-	Reward       [][]*hexutil.Big
-	BaseFee      []*hexutil.Big
-	GasUsedRatio []float64
+type feeHistoryResult struct {
```

Comment:
Also my mistake, but the field names have been changed in the spec last minute which I forgot to update in the code:
"firstBlock" is now called "oldestBlock" and "baseFee" is now called "baseFeePerGas".
https://github.com/ethereum/eth1.0-specs/blob/master/json-rpc/spec.json#L246

Reply:
Fixed, PTAL